src/desktop/components/search_bar/view.coffee

```diff
@@ -134,7 +151,11 @@ module.exports = class SearchBarView extends Backbone.View
 
     @spotlightSearchResult = searchResult
     currentQuery = @$input.val()
-    remainingText = searchResult.get('display').replace(new RegExp(currentQuery, 'i'), '')
+    if @isSubsequentMatch
```

Comment:
For display, if we don't have an exact leading match (the existing scenario), we just display an interpunct and the whole search result. The exact leading match auto-completes for you, that's unchanged.

Reply:
do we need to manually trim the whole search result if it hits the boundary of the search box in this case? what does that look like? i'm thinking mainly in the header search bar which is not as wide... this may be a frequent occurrance.

Reply:
You're right - the length heuristic probably has us covered most of the time for long names. So maybe there's just some corner cases where people type in long queries on long matches, but that is probably very uncommon - I wouldn't worry too much about it in that case!